.gitignore

```diff
@@ -3,6 +3,8 @@
 *.pyc
 *.1-r
 .*.kak.*
+.ccls-cache
+compile_commands.json
```

Comment:
These two seem to be added by mistake?

Reply:
Actually, it was intentional, but I will remove those from the main .gitignore and put them `.git`.

Reply:
Right, I meant adding them to this PR.